README.md

```diff
@@ -1,14 +1,12 @@
-This project name is "Mastering the Art of Becoming a Beggar: A 7-Day Guide to Loose Everything" 
+This project name is "Mastering the Art of Becoming a Beggar: A 7-Day Guide to Loose Everything"
```

Comment:
Correct the typo in the project description. Change "loose" to "lose". [important]